app/presenters/outcome_presenter.rb

```diff
@@ -31,40 +55,60 @@ def has_calendar?
     calendar.present?
   end
 
-  def has_body?
-    use_template? || super()
-  end
-
   def body
-    if use_template?
-      view_context = @state.dup
```

Comment:
I suspect we wanted to make sure the state object couldn't be modified. It might be possible to achieve that another way e.g. hiding the "setter" methods on the instance of `State`.